tests/spread/basic/task.yaml

```diff
@@ -0,0 +1,33 @@
+summary: Ensure that things worked
+
+execute: |
+  # for various utilities
+  . "$EXTTESTSLIB/nested.sh"
+
+  # Start the nested UC20 VM
+  start_nested_core_vm_unit
+
+  # At this point we are able to SSH to the nested VM, so things probably worked
+  # but we should wait for snapd to finish seeding anyways, but first we need to
+  # wait for the snap command to exist
+  nested_wait_for_snap_command
```

Comment:
I would add until this line in a prepare phase like this:

prepare: |
  # for various utilities
  . "$EXTTESTSLIB/nested.sh"
  # Start the nested UC20 VM
  start_nested_core_vm_unit
  # At this point we are able to SSH to the nested VM, so things probably worked
  # but we should wait for snapd to finish seeding anyways, but first we need to
  # wait for the snap command to exist
  nested_wait_for_snap_command